Networking/Networking/Remote/SiteRemote.swift

```diff
@@ -189,6 +218,32 @@ public extension SiteCreationResponse {
 }
 
 /// Answers from the site creation profiler questions.
+public struct StoreProfilerAnswers: Codable {
```

Comment:
I added a new model in order to remove the `SiteProfilerData` later after releasing this optimize profiler project. 
- `SiteProfilerData` requires store name.
- The `SellingStatus` seemed to have different values. 

Based on your [other comment](https://github.com/woocommerce/woocommerce-ios/pull/10459#discussion_r1290856704) I will check whether I can reuse `SiteProfilerData`

Reply:
Can we maybe use the existing model and add an item to remove store name from it instead? I'm also not sure if we need the model to conform to `Codable` as we don't send it directly to the API requests.

Reply:
I have to store this struct into user defaults in a future PR. I decided to stick to using `StoreProfilerAnswers` to avoid adding complexity. 
I will remove the `SiteProfilerData` and its usage in future PRs.